tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -40,7 +41,7 @@ def test_extract_documentation_C(self):
                  " This is the main function.\n"
                  "\n"
                  " @returns Your favorite number.\n"),
-                "C", "doxygen",
+                "C", "doxygen", "",
```

Comment:
Well if I dont put it, then I have to pass in everything as keyword arguments right?